### DISCUSSION TITLE
Setting page size/paper type

### DISCUSSION THREAD

**Discussion by rikaye** [dolanmiu/docx#903] [Q&A] [answered]
How can I set the page size of a document (i.e.; A3/4 vs Letter vs Legal, etc.)?

TIA

**Comment by dolanmiu** [comment, marked as answer]
Check out the demo here:

https://github.com/dolanmiu/docx/blob/master/demo/65-page-sizes.ts

**Reply by rikaye** [reply]
Thanks! I'll check that out.
